<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timetable</class>
 <widget class="QDialog" name="Timetable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_timetable">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>651</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_date2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DATA SPOTKANIA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_date2"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>UCZESTNIK SPOTKANIA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_notes_meet2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>NOTATKI DO SPOTKANIA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_notes2"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb_add_meet2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>391</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DODAJ SPOTKANIE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_update_meet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>391</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>EDYTUJ SPOTKANIE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2_id_timetable">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_delete_meet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>391</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>USUÅƒ SPOTKANIE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
